jsdocConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "opts": {
+    "destination": "./docs"
```

Comment:
I'd like to see doc under the dist folder, similar to the ngDocs of angular-patternfly.